api/CcsSso.Core.Service/External/OrganisationProfileService.cs

```diff
@@ -659,10 +663,6 @@ public async Task UpdateOrganisationEligibleRolesAsync(string ciiOrganisationId,
 
         // Remove service client id inmemory cache since role update
         _localCacheService.Remove($"ORGANISATION_SERVICE_CLIENT_IDS-{ciiOrganisationId}");
-        
-        // Notify the adapter
-        await _adapterNotificationService.NotifyOrganisationChangeAsync(OperationType.Update, ciiOrganisationId);
-
```

Comment:
I would like to know why the notification call removed from the logic. Is it intentional?